ocp_resources/storage_class.py

```diff
@@ -102,3 +104,13 @@ def to_dict(self):
                 self.res.update({"allowedTopologies": self.allowed_topologies})
             if self.mount_options:
                 self.res.update({"mountOptions": self.mount_options})
+
+    @property
+    def storage_profile(self):
+        try:
+            return StorageProfile(
+                client=self.client,
+                name=self.name,
+            )
+        except ResourceNotFoundError:
```

Comment:
Based on what i understand, `StorageProfile` must exist for storage class. please raise if not found

Reply:
done